src/index.html

```diff
@@ -7,6 +7,7 @@
     <meta key='og:site_name' property='og:site_name' content='Syncano' />
     <meta key='og:description' property='og:description' content='Don’t rebuild what has already been built. Syncano has hundreds of ready to use functions for your backend. Set up a scalable app in the cloud in just 45 seconds.' />
     <meta key='og:type' property='og:type' content='website' />
+    <script src=“https://cdn.optimizely.com/js/8415101584.js“></script>
```

Comment:
Why did you add optimizely? It's already on devel https://github.com/Syncano/syncano.com/commit/94616ca159ddb37feebe7de0e05ea2be7de95755

Also you used bad quotation mark.